admin/app/model/AdminLifecycle.scala

```diff
@@ -10,13 +10,18 @@ import conf.switches.Switches._
 import football.feed.MatchDayRecorder
 import jobs._
 import play.api.inject.ApplicationLifecycle
-import play.api.libs.ws.WSClient
 import services.EmailService
 import tools.{AssetMetricsCache, CloudWatch, LoadBalancer}
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class AdminLifecycle(appLifecycle: ApplicationLifecycle, jobs: JobScheduler, akkaAsync: AkkaAsync, emailService: EmailService, wsClient: WSClient)(implicit ec: ExecutionContext) extends LifecycleComponent with Logging {
+class AdminLifecycle(appLifecycle: ApplicationLifecycle,
+                     jobs: JobScheduler,
+                     akkaAsync: AkkaAsync,
+                     emailService: EmailService,
+                     fastlyCloudwatchLoadJob: FastlyCloudwatchLoadJob,
+                     r2PagePressJob: R2PagePressJob,
+                     videoEncodingsJob: VideoEncodingsJob)(implicit ec: ExecutionContext) extends LifecycleComponent with Logging {
```

Comment:
[nitpick] I usually like when parameters are indented like that
```scala
class AdminLifecycle(
  appLifecycle: ApplicationLifecycle,
  jobs: JobScheduler,
  akkaAsync: AkkaAsync,
  emailService: EmailService,
  fastlyCloudwatchLoadJob: FastlyCloudwatchLoadJob,
  r2PagePressJob: R2PagePressJob,
  videoEncodingsJob: VideoEncodingsJob
)(implicit ec: ExecutionContext)
```
that's entirely aesthetic no need to actually fix that.